src/main/java/seedu/address/logic/parser/Parser.java

```diff
@@ -102,6 +103,7 @@ private Command prepareAdd(String args){
             return new AddCommand(
                     matcher.group("name"),
                     matcher.group("phone")==null?" ":matcher.group("phone"),
+                    matcher.group("period")==null?"23:59":matcher.group("period"),
```

Comment:
I understand the constraint with natty. But can we still pass in an empty string, but have a layer of check for empty string before passing to natty? If empty string then don't let it be processed by natty.